etc/signal-cli.profile

```diff
@@ -0,0 +1,49 @@
+# Firejail profile for signal-cli
+# Description: signal-cli provides a commandline and dbus interface for signalapp/libsignal-service-java
+# This file is overwritten after every install/update
+# Persistent local customizations
+include signal-cli.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+
+noblacklist ${HOME}/.local/share/signal-cli
```

Comment:
Add `blacklist ${HOME}/.local/share/signal-cli` to `disable-programs.inc`.

Reply:
Done